paddlets/automl/autots.py

```diff
@@ -92,20 +92,20 @@ def __init__(
             estimator: Union[str, Type[BaseModel], List[Union[str, Type[BaseTransform], Type[BaseModel]]]],
             in_chunk_len: int,
             out_chunk_len: int,
-            skip_chunk_len: int = 0,
-            sampling_stride: int = 1,
-            search_space: Union[str, dict] = 'auto',
-            search_alg: str = 'auto',
-            resampling_strategy: str = 'auto',
-            split_ratio: Union[str, float] = 'auto',
-            k_fold: Union[str, int] = 'auto',
-            metric: str = 'auto',
-            mode: str = 'auto',
-            refit: bool = True,
-            ensemble: bool = False,
+            skip_chunk_len: Optional[int] = 0,
+            sampling_stride: Optional[int] = 1,
+            search_space: Optional[Union[str, dict]] = 'auto',
+            search_alg: Optional[str] = 'auto',
+            resampling_strategy: Optional[str] = 'auto',
+            split_ratio: Optional[Union[str, float]] = 'auto',
+            k_fold: Optional[Union[str, int]] = 'auto',
+            metric: Optional[str] = 'auto',
+            mode: Optional[str] = 'auto',
+            refit: Optional[bool] = True,
+            ensemble: Optional[bool] = False,
             local_dir: Optional[str] = None,
-            n_jobs: int = -1,
-            verbose: int = 4
+            n_jobs: Optional[int] = -1,
+            verbose: Optional[int] = 4
```

Comment:
Line95 - Line105 and Line107 - Line108 should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional